redaxo/src/addons/project/package.yml

```diff
@@ -2,3 +2,5 @@ package: project
 version: dev
 author: Project Admin
 supportpage: www.redaxo.org/de/forum/
+
+load: late
```

Comment:
ggf. hier auch nochmal ein teil des kommentars hinpacken, damit das niemand in sein addon übernimmt?

Reply:
habs kommentiert, nehme wie immer sehr gerne bessere formulierungs-vorschläge an.

Reply:
passt.

was mir dabei auffällt. vllt sollte man (separater PR) im project addon mal eine README hinterlegen und grob beschreiben wozu dieses gedacht ist..

Reply:
Jupp stimmt, wäre sinnvoll!